protocol_driver_grpc.cc

```diff
@@ -96,6 +96,7 @@ void ProtocolDriverGrpc::SetNumPeers(int num_peers) {
 }
 
 ProtocolDriverGrpc::~ProtocolDriverGrpc() {
+  ShutdownClient();
   ShutdownServer();
   grpc_client_stubs_.clear();
```

Comment:
this line of code should be inside shutdown client, and I think we should be calling shutdown server before shutdown client.

Reply:
Done